src/workerd/api/tests/js-rpc-test.js

```diff
@@ -1179,11 +1197,30 @@ export let crossContextSharingDoesntWork = {
 
 export let waitUntilWorks = {
   async test(controller, env, ctx) {
-    globalWaitUntilPromise = null;
-    await env.MyService.testWaitUntil();
+    // Tests ctx.waitUntil
+    {
+      globalWaitUntilPromise = null;
+      await env.MyService.testWaitUntil();
+
+      assert.ok(globalWaitUntilPromise instanceof Promise);
+      await globalWaitUntilPromise;
+    }
+
+    // Tests `import { waitUntil } from 'cloudflare:workers` on WorkerEntrypoint
+    {
+      globalWaitUntilPromise = null;
+      await env.MyService.testImportedWaitUntil();
 
-    assert.strictEqual(globalWaitUntilPromise instanceof Promise, true);
-    await globalWaitUntilPromise;
+      assert.ok(globalWaitUntilPromise instanceof Promise);
+      await globalWaitUntilPromise;
+    }
+
+    // Tests `import { waitUntil } from 'cloudflare:workers` directly
+    {
+      const { promise, resolve } = Promise.withResolvers();
+      waitUntil(scheduler.wait(100).then(resolve));
+      await promise;
```

Comment:
This doesn't test anything interesting. This promise would resolve even without the `waitUntil()` call above. `waitUntil()` only makes a difference if the promise is still running when the event ends, but that's not the case here.